### PR TITLE
Update dbot.md

### PR DESCRIPTION
**Pull request by Bargenish**
Added the reliability field and it's approved values

## Status
In Progress

## Related Issues
relates: https://github.com/demisto/content/pull/10837

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/dbot.md [hunk 1]
```diff
@@ -11,19 +11,35 @@ DBot is the Cortex XSOAR machine learning bot which ingests information about in
...
-| Type | Can be: ip, file, email, or url|
-| Vendor | This is the vendor reporting the score of the indicator|
-| Score | An int representing the status of the indicator. See Score Types below|
-
+The DBot score must be at the root level of the context and contain **all** the required keys as listed below.
+
+| Key | Meaning | Required
+| --- | --- | --- |
+| Indicator | Can be: IP, SHA1, MD5, SHA256, Email, or Url | Required |
+| Type | Can be: ip, file, email, or url | Required |
```
Comment (Bargenish) [inline docs/integrations/dbot.md:24]: @glicht Should I replace the values to match the DBotScoreType?

Comment (glicht) [reply docs/integrations/dbot.md:24]: You can

File: docs/integrations/context-standards-mandatory.md [hunk 2]
```diff
@@ -573,6 +574,9 @@ The following is the format for a DBot Score entry.
 - contextPath: DBotScore.Score
   description: The actual score.
   type: Number
+- contextPath: DBotScore.Reliability
+  description: Reliability of the source providing the intelligence data.
+  type: Number
```
Comment (fvigo) [inline docs/integrations/context-standards-mandatory.md:579]: This should be String?

**Comment by fvigo** [comment]
How can we support this if it's not implemented in CommonServerPython? https://github.com/demisto/content/blob/f073fbc39b274b2fe98ef6a1fd156eb3f3e2d9bc/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L2135

**Comment by Bargenish** [comment]
@fvigo We have a PR ready(https://github.com/demisto/content/pull/10837). This will be merged after.

**Comment by kirbles19** [comment]
@Bargenish Done

**Review by yaakovi — APPROVED** [review]

**Pull request merged**